pkgs/misc/game-rs/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, steam-run
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "game-rs";
+  version = "0.1.3";
+
+  src = fetchFromGitHub {
+    owner = "amanse";
+    repo = "game-rs";
+    rev = "refs/tags/v${version}";
```

Comment:
```suggestion
    rev = "v${version}";
```

Reply:
i'll try this, sorry life got in way...

Reply:
I'm still getting the same cargo hash as before, if i use hash from @wozeparrot the build fails, sorry for using your time, but i don't know what to do now lol, you can close the request if you wish.

Reply:
> if i use hash from @wozeparrot the build fails, sorry for using your time

Is that due to hash mismatch? Because I've been getting the same hash.

Reply:
yes, it is due to hash mismatch
https://github.com/NixOS/nixpkgs/pull/218590#issuecomment-1619115145 this but specified and got are reversed if i use their provided hash
 
>Getting the same hash

as me? or as them?

Reply:
As them.

Reply:
can you try building now please? I think it's codespaces vm messing something up, I tried locally and got the same hash error as you both.

Reply:
I'll try building on a proper azure vm, maybe codespaces doesn't clear /nix/store properly with the permission and their weird fs

Reply:
build succeeded on the the commit i pushed just now, got the same vendor hash as you two, it was codespaces shenanigans then ig, thanks for the patience!